src/bench/merkle_root.cpp

```diff
@@ -2,11 +2,11 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
-#include "bench.h"
+#include <bench.h>
```

Comment:
This doesn't compile locally here. Should be `#include <bench/bench.h>`

Reply:
Fixed!